client/src/test/index.browser.ts

```diff
@@ -7,6 +7,7 @@ export function run(): Promise<void> {
       ui: 'tdd',
       reporter: undefined
     });
+    mocha.timeout(100000);
```

Comment:
Is 100 seconds a bit excessive?

Reply:
Yeah, fair point. I just copied the value from the original node test suite. Test suite takes about 4~5 seconds to run right now. Each tests suite adds about 2-3 seconds to execute. We'd still be pretty safe, with room for growth, changing both values to ~30 seconds.

What are you thinking?

Reply:
Let me adjust this and fix the launch.config formatting and wording.